templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -30,15 +30,13 @@ contents:
            exit 1
         fi
       else
-        GOOGLE_URL=https://storage.googleapis.com/etcd
-        DOWNLOAD_URL=${GOOGLE_URL}
-
-        echo "Downloading etcdctl binary.."
-        curl -s -L ${DOWNLOAD_URL}/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -o $ASSET_DIR/tmp/etcd-${ETCD_VERSION}-linux-amd64.tar.gz \
-          && tar -xzf $ASSET_DIR/tmp/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -C $ASSET_DIR/shared --strip-components=1 \
-          && mv $ASSET_DIR/shared/etcdctl $ASSET_DIR/bin \
-          && rm $ASSET_DIR/shared/etcd \
-          && $ETCDCTL_BIN version
+        local etcdimg="{{.Images.etcdKey}}"
+        local etcdctr=$(podman create ${etcdimg})
```

Comment:
is it a fair assumption that the etcd image will always be available and doesn't need to be pulled ?

Reply:
Should be as this is a master node the relationship is 1:1